netmanager/src/views/components/Sites/SiteToolbar.js

```diff
@@ -224,13 +224,13 @@ const SiteToolbar = (props) => {
       const activeNetwork = JSON.parse(localStorage.getItem('activeNetwork'));
 
       if (!isEmpty(activeNetwork)) {
-        dispatch(loadSitesData(activeNetwork.net_name));
+        dispatch(loadSitesData(activeNetwork.net_name || activeNetwork.grp_title));
       }
 
       dispatch(
         updateMainAlert({
           message: `${resData.message}. ${
-            siteData.network !== activeNetwork.net_name
+            siteData.network !== activeNetwork.net_name || activeNetwork.grp_title
```

Comment:
We shall need to revisit this logic, this might be the cause of some problems. We cannot assign a grp_title and call it a network!! @OchiengPaul442

Reply:
@OchiengPaul442 , i was hoping for something like this along the lines of this pseudocode below:
```
let key = "";
let value = "";
value = [activeOrganisation.net](http://activeorganisation.net/)_name || activeOrganisation.grp_title;
if (activeOrganisation.grp_title) {
  key = "group";
}
if ([activeOrganisation.net](http://activeorganisation.net/)_name) {
  key = "network";
}
//api_call?[key]=value
```